src/app/[locale]/_component/noSessionActiveComp/noSessionActiveComp.tsx

```diff
@@ -40,7 +49,7 @@ const NoSessionActiveComp = ({ title }: NoSessionProps): React.ReactElement => {
         </Grid>
         <Grid item xs={12}>
           <Button onClick={() => handleCloseBtn()} sx={{ mr: 2 }} variant="outlined">
-            {t('common.backtohome')}
+            {isL1 ? t('common.close') : t('common.backtohome')}
```

Comment:
This logic create a glitch, as you can see into the screen recording. If the user is L1, after click button, the button text change. And as you can see into the screen record there are some problems on the console, some warnings, but the warning depends by some different pages

https://github.com/pagopa/io-web-profile/assets/83651704/8b3e28dd-c1fa-48d4-a2c2-f0bfe66dbf48

Reply:
warnings in the console after the build are no longer shown